workbench/src/renderer/components/InvestTab/index.jsx

```diff
@@ -226,6 +237,8 @@ export default class InvestTab extends React.Component {
       return (<div />);
     }
 
+    // Alert won't expire during archiving; will expire 2s after completion
+    const alertExpires = (saveAlert === 'archiving...') ? 1e7 : 2000;

```

Comment:
Is the `1e7` just an arbitrary length of time to wait until this `delay` function is called again?

Reply:
Arbitrarily long, yes. ~167 minutes. The idea is to never expire the "archiving..." message, but change it to "completed" whenever it's done, and then display that for 2 more seconds.